test/integration/included/stanc.expected

```diff
@@ -1,41 +1,47 @@
   $ ../../../../install/default/bin/stanc --include_paths="../included" --auto-format badrecurse1.stan
 
 Syntax error at file ../included/badrecurse1.stan, line 1, column 0, included from
-file ../included/badrecurse1.stan, line 1, column 25, included from
-file badrecurse1.stan, line 1, column 25, includes error:
+file ../included/badrecurse1.stan, line 1, column 0, included from
+file ../included/badrecurse1.stan, line 1, column 0, included from
+file badrecurse1.stan, line 1, column 0, includes error:
    -------------------------------------------------
      1:  #include badrecurse1.stan
          ^
    -------------------------------------------------
 
 Found cyclical include structure.
```

Comment:
[optional]

I think the error would be clearer as 

```
File 'badrecurse1.stan' recursively included itself.
```

if it's not too much work.  I just think "cyclical include structure" might be too opaque.  It makes the user go back and inspect the includes to see that badrecurse1.stan included itself.

Reply:
Done.